pages/common/xml-elements.md

```diff
@@ -0,0 +1,28 @@
+# xml-elements
+
+> Extract elements and structure of an XML document.
+> More information: <http://xmlstar.sourceforge.net/docs.php>.
+
+- Extract elements of an XML document (as XPATH expressions):
```

Comment:
Perhaps this will read better?
```suggestion
- Extract elements from an XML document (as XPATH expressions):
```

Reply:
xml-elements.md:- Extract elements from an XML document (producing XPATH expressions):